packages/workflow/src/NodeErrors.ts

```diff
@@ -1,6 +1,6 @@
+/* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
```

Comment:
we should really be trying to get rid of file level linting exceptions, and add the on a line by line basis.